backend/db/migrate/20220808082718_changes_for_open_calls.rb

```diff
@@ -28,10 +28,13 @@ def change
   private
 
   def migrate_existing_open_calls!
-    country = Location.find_by! location_type: :country, name_en: "Colombia"
     OpenCall.all.each do |open_call|
       open_call.assign_attributes country_id: country.id, instrument_types: [open_call.instrument_type]
       open_call.save validate: false
     end
   end
+
+  def country
+    @country ||= Location.find_by! location_type: :country, name_en: "Colombia"
```

Comment:
just slight tweak of migration --> deployment runs migrations and it fails on empty db which does not have `Colombia` location. This fix will ask for Colombia location only when there are already some Open Calls.